rfc/README.md

```diff
@@ -0,0 +1,42 @@
+# The OpenTofu RFC process
+
+If a larger body of work is needed for an enhancement in the [main OpenTofu repository](https://github.com/opentofu/opentofu) or other OpenTofu repositories, the core team often asks for an RFC to be written on how a feature should be designed and implemented. We do this in order to make sure everyone is on the same page how a feature should be implemented. This repository holds past RFCs and anyone can contribute RFCs.
+
+We generally recommend [opening an enhancement issue](https://github.com/opentofu/opentofu/issues/new/choose) before going through the effort of writing an RFC. Not all enhancements require an RFC, but the core team may call for an RFC to be written by adding the `needs-rfc` label.
+
+## Filing an RFC
+
+Filing and getting an RFC merged is intentionally a longer process. We use pull requests to fully define and discuss technical solutions to a feature request. 
+
+Before filing an RFC, ideally open an enhancement issue in the [main repository](https://github.com/opentofu/opentofu) in order to discuss if the feature has enough community support to warrant working on an RFC. This step is optional, but highly recommended.
+
+To file an RFC, please create a PR by copying the contents of the [template](template/) into a new folder. The name of the folder does not matter at this stage, the core team will assign it an RFC number when it is accepted.
```

Comment:
Let's let people be lazy and give them a "newrfc.sh" which will make this for them.